agents/kubevirt/fence_kubevirt.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python -tt
+
+import os
+import sys, re
+import logging
+import atexit
+sys.path.append("/usr/share/fence/")
+from fencing import *
+from fencing import fail, fail_usage, run_delay, EC_STATUS, SyslogLibHandler
+from kubernetes import client, config
+from openshift.dynamic import DynamicClient
+from kubernetes.client.exceptions import ApiException
+
+API_VERSION='kubevirt.io/v1'
+
+logger = logging.getLogger("fence_kubevirt")
```

Comment:
You should use logging from the fencing library to make it work like the other agents. You can use -v or verbose=1 or verbosity-level=X to make it show more info. `fence_gce` has an example of how to add more handlers, but here it seems like you only need the botocore part.

Reply:
Fixed